environments/environments.py

```diff
@@ -54,11 +54,11 @@ def _draw(self):
         """
         raise NotImplementedError
```

Comment:
Not necessary to raise a `NotImplementedError` in `abstractmethod`s. The `ABC` superclass will already raise the correct exception by itself if a class that does not implement the method is instantiated. You can just use pass
```suggestion
        pass
```

Reply:
Okay, I just took a placeholder code haha.